tests/command/test_undo.rs

```diff
@@ -455,7 +455,7 @@ fn test_historical_smartlog_visibility() -> eyre::Result<()> {
         insta::assert_snapshot!(screen_to_string(&screenshot1), @r###"
```

Comment:
If you look at the line above this one (it seems GitHub doesn't let me comment on it because it's not modified), you'll see the answer to why this passes locally and fails on CI: CI's git version apparently does not support reference transactions. So it's probably a good idea if you can find a git version that doesn't and run the tests locally with that version as well. The CI logs indicate that it's using git-v2.24.3, so try installing that and pointing `TEST_GIT` and `TEST_GIT_EXEC_PATH` to it.